lms/djangoapps/django_comment_client/utils.py

```diff
@@ -154,17 +155,19 @@ class DiscussionIdMapIsNotCached(Exception):
     pass
 
 
-def get_cached_discussion_key(course, discussion_id):
+@request_cached
+def get_cached_discussion_key(course_id, discussion_id):
```

Comment:
Out of curiosity, is this something you saw as a performance issue?  I can see moving from RequestCache to decorator for code cleanliness, but am curious about how beneficial you are finding it on calls like this?